Sources/SkyFloatingLabelTextField.swift

```diff
@@ -31,17 +31,17 @@ open class SkyFloatingLabelTextField: UITextField {
     // MARK: Animation timing
 
     /// The value of the title appearing duration
-    open var titleFadeInDuration:TimeInterval = 0.2
+    dynamic open var titleFadeInDuration:TimeInterval = 0.2
     /// The value of the title disappearing duration
-    open var titleFadeOutDuration:TimeInterval = 0.3
+    dynamic open var titleFadeOutDuration:TimeInterval = 0.3
```

Comment:
I haven't done any UIAppearance customization with Swif, but isn't there a need for a marking such as `UI_APPEARANCE_SELECTOR` in Objective-C to inform the run time that this can be customized via UIAppearance?